modules/game/src/main/GameRepo.scala

```diff
@@ -112,7 +114,7 @@ object GameRepo {
       case (Nil, Nil) => funit
       case (sets, unsets) => coll.update(
         $id(progress.origin.id),
-        nonEmptyMod("$set", $doc(sets)) ++ nonEmptyMod("$unset", $doc(unsets))
+        nonEmptyMod("$set", $doc(sets)) ++ nonEmptyMod("$unset", $doc(unsets: _*))
```

Comment:
Deprecation warning (from the variant of `$doc` using deprecated `BSONDocument` factory)